### ISSUE TITLE
Update to the latest CoC

### ISSUE THREAD

**Issue by krainboltgreene** [hamstergem/hamster#205]
We're behind a bit: http://contributor-covenant.org/

**Comment by robinboening** [comment]
Just picked this issue for the upcoming 9th edition of https://rubyissues.ongoodbits.com :)

**Comment by studiolo** [comment]
Hi, I saw this on Ruby Issues and would like to call dibs - to support the good cause and to practice git bit.

**Comment by krainboltgreene** [comment]
Awesome have at it :)

**Comment by studiolo** [comment]
The new CoC version asks for an email address where people can report incidents to the project maintainers. Please advise.

**Comment by studiolo** [comment]
The new CoC version has additional clauses, listed below. Could I ask the project maintainers to discuss these and to let me know whether they accept them? Then I'll update the CoC file and send a pull request.

* "Publishing other's private information, such as physical or electronic addresses, without explicit permission" is considered harassment.
* Project maintainers can ban contributors that violate the CoC or exhibit "other behaviors that they deem inappropriate, threatening, offensive, or harmful", temporarily or permanently.
* " Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team."
* "This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community."

**Comment by krainboltgreene** [comment]
me@kurtisrainboltgreene.name, I take personal responsibility.

I appreciate the details, but we definitely agree with it.

**Comment by krainboltgreene** [comment]
@studiolo You're right, follow this example: https://github.com/krainboltgreene/blankgem.gem

**Comment by studiolo** [comment]
I think I've got it now: https://github.com/hamstergem/hamster/pull/209
